### PR TITLE
Inventory

### PR DESCRIPTION
**Pull request by Flryang**
Added 2d assets for the inventory

### CODE REVIEW

File: Assets/Scripts/Inventory.cs [hunk 1]
```diff
@@ -22,33 +23,15 @@ public ItemPickupResult PickupItem( ref PickupItem item )
...
+        {
+            int i = (int) Math.Ceiling(slot / 2.0); 
+            int pos = (int) (cursor + i * Math.Pow(-1.0, slot) + inventorySize) % inventorySize; 
+            if (m_items[pos] == null){
+                m_items[pos] = item;
                 item.OnPickup();
-                inventoryDisplay.addItem(item, pos1);
+                inventoryDisplay.addItem(item, pos);
+                inventoryDisplay.showItemName(item.itemName);
                 PlayerController.PlaySound( "object_obtained" );
```
Comment (ZuhabWasim) [inline Assets/Scripts/Inventory.cs]: These will be changed to incorporate Lakshya's new AudioPlayer instead

**Review by ZuhabWasim — CHANGES_REQUESTED** [review]
Looks good Meilin! There will be a few changes you'll need to implement following Lakshya's overhaul of the audio system so once that's merged, let's incorporate those changes and have this merged too.

Also for the image files, would you be able to rename them to follow the naming conventions recently in [guidelines.md](https://github.com/ZuhabWasim/ReflectionGame/blob/main/guidelines.md)? So instead of `Inventory_Gear (Unmoving Piece) FINAL.png` we'd have `inventory_gear_unmoving_piece.png`

**Review by ZuhabWasim — APPROVED** [review]
Looks good to me. Looks like the Camera Mirror Position script was not attached to the mirrors as before so we added it back to the prefab. Looks like it's fine now

Additionally, looks like the only error observable is that copy-pasting the present world copied the script for the vanity mirror so both the past and present are caching an audio source at the same key, causing the audio to not play. Solution, disable the script in VanityMirrorPast

**Comment by Flryang** [comment]
Somehow me and Zuhab cannot rebase and merge this branch to main. Could you please take a look at this and see if you are able to rebase? Thanks @Lakshya2610 @dargonfyl

**Comment by dargonfyl** [comment]
GitHub says that it can automatically merge, but I'm going to try to rebase for now.

**Review by dargonfyl — APPROVED** [review]

**Pull request merged**